general/standards/electrical-software-quality-management.rst

```diff
@@ -1,2 +1,22 @@
 Electrical & Software Quality Management Process
 ================================================
+
+PCB designs
+-----------
+
+* Minimum 4 approves for schematics review
```

Comment:
This is a bit much no? This was already a huge bottleneck last year to the point where it got ignored.